### ISSUE TITLE
"Push Commits" window to large for 13' screen & no scroll

### ISSUE THREAD

**Issue by luishocke** [ChrisRM/material-theme-jetbrains#1453]
### Summary
The "Push Commits" window is to large for a 13' screen & you can't scroll, leaving the "push" and "commit" buttons outside the screen / below the windows taskbar.
...

### Expected behavior
Window with "Push" and "Cancel" Button

### Actual behavior
Both buttons are outside the viewable part of the window. You can't scroll or move the window further upwards, as it takes already the whole screen. The window can be extended by hiding the windows taskbar and dragging the bottom border, but the buttons will then be behind the taskbar when toggling it on again.

### Steps to reproduce (if needed)
[Screen: 13.3inch, 16:9, 1920x1080px]
1. Click on Commit
2. Click on Commit and Push

### Environment
[Screen: 13.3inch, 16:9, 1920x1080px]

###Screenshot
<img width="960" alt="2019-12-02 15_09_28-" src="https://user-images.githubusercontent.com/22380775/69985404-fb7c5900-1532-11ea-9e14-23ab323e6333.png">

_________________________________________________________
**About**

WebStorm 2019.2.4
Build #WS-192.7142.35, built on October 29, 2019
Licensed to Luis Hocke
Subscription is active until January 16, 2020
For educational use only.
Runtime version: 11.0.4+10-b304.77 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 1998M
Cores: 4
Registry: ide.tree.ui.experimental=false, ide.balloon.shadow.size=0
Non-Bundled Plugins: Statistic, com.chrisrm.idea.MaterialThemeUI, net.vektah.codeglance, izhangzhihao.rainbow.brackets, ru.adelf.idea.dotenv

_____________________________________________________________________________

**Custom Theme:**
```
<application>
  <component name="MaterialCustomThemeConfig">
    <option name="excludedColor" value="0b253aff" />
    <option name="accentColor" value="ff1675ff" />
    <option name="notificationsColor" value="152c3bff" />
    <option name="treeSelectionColor" value="152c3bff" />
    <option name="highlightColor" value="152c3bff" />
    <option name="secondBorderColor" value="2a373eff" />
    <option name="tableSelectedColor" value="152c3bff" />
    <option name="contrastColor" value="001424ff" />
    <option name="disabledColor" value="2e3c42ff" />
    <option name="secondaryBackgroundColor" value="011627ff" />
    <option name="buttonColor" value="2e3c43ff" />
    <option name="selectionForegroundColor" value="ffffffff" />
    <option name="selectionBackgroundColor" value="011627ff" />
    <option name="textColor" value="607d8bff" />
    <option name="foregroundColor" value="b0bec5ff" />
    <option name="backgroundColor" value="011627ff" />
  </component>
</application>
```

**Comment by mallowigi** [comment]
Wow, I have no idea how you got that result... was it always like this or recently?

**Comment by mallowigi** [comment]
Anyway you can resize the window do you ?

**Comment by luishocke** [comment]
> Anyway you can resize the window do you ?
If  I resize the window to be larger, the buttons are below the task bar, so I can't press them. If I make it smaller, there is no scroll in the window and the buttons are not visible. I can only use them by clicking on the small visible part you see in the screenshot or moving the taskbar away.


> Wow, I have no idea how you got that result... was it always like this or recently?
I recently reinstalled Webstorm (update failed and corrupted my installation), I didn't have the problem before. But it's possible that my previous installation didn't have the latest plugin version, so I can't tell whether the problem was caused by the Webstorm or plugin update.

.. .Just updated Webstorm to 2019.3 (and the material UI plugin): 
- The window is now much smaller.
- Resizing works (buttons do not disappear any more if the window is too small)

I reported this in the  [Webstorm Issue Tracker](https://youtrack.jetbrains.com/issue/WEB-42767) first and was told that there is a similar problem with the Gerrit plugin, possibly it was solved on their end now, no idea. I changed nothing else than the updated.
